### PR TITLE
Exercise 1

### PR DESCRIPTION
**Pull request by davepmb**

### CODE REVIEW

**Comment by davepmb** [comment]
I hope I pulled the correct Directory!
I have two on my computer and not sure why I have been having issues with Pulling in Git.

File: templates/index.html [hunk 1]
```diff
@@ -12,6 +12,9 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        Pick up Groceries
```
Comment (Jonesey13) [inline templates/index.html:29]: This is a hardcoded list and not the list of saved todo items from `session_items.py`

File: app.py [hunk 2]
```diff
@@ -4,9 +4,13 @@
...
 app.config.from_object('flask_config.Config')
 
-@app.route('/')
+@app.route('/', methods=['GET', 'POST'])
+
 def index():
-    return 'Hello World!'
+    return render_template('index.html')
+    get_items()
+    add_item(title)
```
Comment (Jonesey13) [inline app.py]: These 2 lines are never executed

File: app.py [hunk 3]
```diff
@@ -4,9 +4,13 @@
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
-@app.route('/')
+@app.route('/', methods=['GET', 'POST'])
+
 def index():
-    return 'Hello World!'
+    return render_template('index.html')
```
Comment (Jonesey13) [inline app.py]: No data is passed to the template

**Review by Jonesey13 — CHANGES_REQUESTED** [review]
There's still some work to be done here:
1. Goal: Display the list of saved todo items
1. Goal: Create new todo items

We can't approve the PR until these goals have been met.

It may be worth a chat to go over these (we can talk about this during today's call).

**Pull request merged**